### ISSUE TITLE
Is Sane Stack dead?

### ISSUE THREAD

**Issue by dskvr** [sane/sane#205]
There doesn't appear to be any recent progress.

**Comment by jamemackson** [comment]
there are a few of us that intend on picking things back up here but so far we've all been swamped with other things for the past few months so things have been at a standstill here for a while. if your interesting in helping with the project any help is appreciated. 😊

**Comment by g5codyswartz** [comment]
I've been looking around trying to take in all the different frameworks trying to choose something for ember to pair up with and this sounds like it should be a great two-in-one but it's also concerning choosing something that has 1-2 years as the last touched timestamp, I'm also now realizing this post is over a year old (I thought it was this year's post for a while)

I'm assuming that I can continue to move forward and still use the latest and greatest with ember and sails while using this.

**Comment by Master244** [comment]
I'm still using this framework as well I love how it is setup also with Docker and having a nice stack to quickly prototype. Though I wouldn't run this in production

**Comment by Globegitter** [comment]
@g5codyswartz Yeah it is unfortunately not being actively developed but if you want to move things forward here I did give you dev access.

**Comment by Master244** [comment]
Ye that would be awesome, at least to get the build passing. I work for Deloitte and I want to introduce this since we are scaffolding a lot this framework would be perfect.

**Comment by dioptre** [comment]
I'm thinking about putting some work into updating this. Anyone interested?

**Comment by jamemackson** [comment]
@dioptre I don't have a lot of extra time these days but am interested in keeping things alive here too.  I have to admit I haven't used sails in a while though (i've switched my api's to [lux](https://github.com/postlight/lux)), do you know the relative health of sails?  Otherwise, I'd advocate swapping sails out for lux, which being a JSON:api fluent framework it's been a better fit over sails, for me anyways.

**Comment by dougestey** [comment]
Sails is alive and well, they've done great work in the 1.0 release (which https://github.com/mdconaway/sails-ember-rest has maintained compatibility with) and I'm fairly happy with where things are headed. I'm interested in Sane stack - would be interested in helping out as well.

Hadn't seen Lux, thanks for the tip - I'll check it out.
